episodes/04-redirection.md

```diff
@@ -236,6 +236,23 @@ $ wc -l SRR098026.fastq
 
 Now you can divide this number by four to get the number of sequences in your fastq file.
 
+This can be done using [shell integer arithmetic](https://www.gnu.org/software/bash/manual/html_node/Shell-Arithmetic.html)
+
+```bash
+$ echo $((996/4))
+```
+
+or [bc - an arbitrary precsion calculator](https://www.gnu.org/software/bc/manual/html_mono/bc.html)
+
+
+```bash
+$ echo "996/4" | bc
+```
+
+```output
+249
+```
+
```

Comment:
`bc` is useful but I think I would avoid introducing another tool as there is already a lot for people to take in

Reply:
Can remove bc. Maybe it is better to use `printf %.3f%% "$((1000 * 996/4))e-3"`
so that some decimal places can be shown?  Integer arithmetic might hide an
error since `echo "$((997/4))"` gives the same answer.

Reply:
I would probably lean towards keeping what you currently have and adding a note that it only handles integer division. I think the syntax for formatting strings with `printf` might be confusing for people that are new to it, and it would probably need some more explanation than we have space for here